awx/ui_next/src/api/Base.js

```diff
@@ -10,6 +10,16 @@ const defaultHttp = axios.create({
   },
 });
 
+defaultHttp.interceptors.response.use(response => {
+  const timeout = response?.headers['session-timeout'];
+  if (timeout) {
+    const timeoutDate = new Date().getTime() + timeout * 1000;
+    window.localStorage.setItem('session-timeout', String(timeoutDate));
```

Comment:
Perhaps we could write like
```suggestion
    window.localStorage.setItem('awx-session-timeout', String(timeoutDate));
   
```
To avoid conflict with other keys in `localStorage`.

Reply:
Done